data-otservbr-global/monster/quests/primal_ordeal_quest/the_primal_menace.lua

```diff
@@ -280,12 +280,13 @@ local function spawnMonster(monsterId, spawnPosition)
 	end
 
 	local randomMonsterIndex = math.random(#thePrimalMenaceConfig.MonsterConfig.MonsterPool)
+	local primalMonster = Game.createMonster(thePrimalMenaceConfig.MonsterConfig.MonsterPool[randomMonsterIndex], spawnPosition)
```

Comment:
You can improve this to make sure the monster exists.
Game.createMonster returns a pointer to Monster*
```suggestion
	local primalMonster = Game.createMonster(thePrimalMenaceConfig.MonsterConfig.MonsterPool[randomMonsterIndex], spawnPosition)
	if not monster then
		logger.error("Cannot create primal monster {}", thePrimalMenaceConfig.MonsterConfig.MonsterPool[randomMonsterIndex])
		return
	end
```